site/src/data/releases/v0.2.json

```diff
@@ -0,0 +1,160 @@
+    "badgeType": "milestone"
+  },
+  "releases": [
+    {
+      "version": "v0.2.0",
+      "date": "June 5, 2025",
+      "type": "minor",
+      "tags": [
+        { "text": "Azure OpenAI Integration", "type": "feature" },
+        { "text": "Sidecar Architecture", "type": "feature" },
+        { "text": "Performance Improvements", "type": "feature" },
+        { "text": "CLI Tools", "type": "feature" },
+        { "text": "Model Failover and Retry", "type": "feature" },
+        { "text": "Certificate Manager Integration", "type": "feature" }
+      ],
+      "overview": "Envoy AI Gateway v0.2.0 builds upon the solid foundation of v0.1.0 with focus on expanding provider ecosystem support, improving reliability and performance through architectural changes, and enterprise-grade authentication support for Azure OpenAI.",
+      "features": [
+        {
+          "title": "Azure OpenAI Integration",
+          "items": [
+            {
+              "title": "Full Azure OpenAI Support",
+              "description": "Complete integration with Azure OpenAI services with request/response transformation for the unified OpenAI compatiple completions API."
+            },
+            {
+              "title": "Upstream Authentication for Azure Enterprise Integration",
+              "description": "Support for accessing Azure via OIDC tokens and Entra ID for enterprise-grade authentication for secure and compliant upstream authentication."
+            },
+            {
+              "title": "Enterprise Proxy URL Support for Azure Authentication",
+              "description": "Enhanced Azure authentication with proxy URL configuration options for enterprise proxy support."
+            },
+            {
+              "title": "Flexible Token Providers",
+              "description": "Generalized token provider architecture supporting both client secret and federated token flows"
+            }
+          ]
+        },
+        {
+          "title": "Architecture Improvements",
+          "items": [
+            {
+              "title": "Sidecar + UDS External Processor",
+              "description": "Switched to sidecar deployment model with Unix Domain Sockets for improved performance and resource efficiency"
+            },
+            {
+              "title": "Enhanced ExtProc Limits",
+              "description": "Configurable external processor resource limits for better resource management"
+            },
+            {
+              "title": "Certificate Manager Integration",
+              "description": "Enabled cert-manager for mutating webhook server for improved security"
+            }
+          ]
+        },
+        {
+          "title": "Cross-Backend Failover and Retry",
+          "items": [
+            {
+              "title": "Provider Fallback Logic",
+              "description": "Automatic failover between AI providers when primary provider is unavailable, ensuring high availability and fault tolerance."
```

Comment:
We just merged the priority based fallback so it is beyond primary/secondary, as endpoints for the highest priority level become unhealthy, traffic will begin to trickle to lower priority levels.